Samples/WindowsCopilotRuntime/cpp-console-unpackaged/CMakePresets.json

```diff
@@ -0,0 +1,61 @@
+{
+    "version": 2,
```

Comment:
If you decide to go with CMake 3.21 as your minimum, you can make this a little cleaner using version "3" instead.

With vesrsion 3, the cacheVariables `CMAKE_INSTALL_PREFIX` and `CMAKE_TOOLCHAIN_FILE` have a better syntax:

```
    {
      "name": "arm64-debug",
      "generator": "Ninja",
      "binaryDir": "${sourceDir}/out/build/${presetName}",
      "architecture": {
        "value": "arm64",
        "strategy": "external"
      },
      "installDir": "${sourceDir}/out/install/${presetName}",
      "toolchainFile": "$env{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake",
      "cacheVariables": {
        "CMAKE_BUILD_TYPE": "Debug"
      }
```